api/utils/queryset_to_structures.py

```diff
@@ -100,21 +100,23 @@ def convert_to_single_dict(queryset, query_type="population"):
 def merge_comparable_querysets(
     queryset_population, queryset_gdp_per_capita, parameter1_type, parameter2_type
 ):
-    sorted_population_list = sorted(
+    sorted_parameter1_list = sorted(
         queryset_population, key=itemgetter("year", "country")
     )
-    sorted_gdp_per_capita_list = sorted(
+    sorted_parameter2_list = sorted(
         queryset_gdp_per_capita, key=itemgetter("year", "country")
     )
+    len_years = min(len(sorted_parameter1_list),len(sorted_parameter2_list))
```

Comment:
Why is this necessary?
Can't we make sure that two querysets are of same size.

(If we use union, then?)